### DISCUSSION TITLE
Fresh install ubuntu 20.04 container stuck at wait_start

### DISCUSSION THREAD

**Discussion by konceptnet** [Security-Onion-Solutions/securityonion#7302] [Unsupported Versions] [answered]
I install Security Onion on a fresh install of Ubuntu 20.04 according with the documentation.
I choose production option and all default, everything was working at the end and before I do the reboot at the end. I was able to receive alert, my VM memory was used  at 95% percent 20GB on 22GB.

But after the reboot , all my container stuck at WAIT_START except for aptcacherng, so-dockerregistry and so-kratos

Can someone help me with my situation please ?

![image](https://user-images.githubusercontent.com/53401670/155376533-a8ecd1f7-f817-48b9-8774-8f813adf006d.png)

**Comment by dougburks** [comment, marked as answer]
I just installed on Ubuntu 20.04, rebooted, and everything came up properly for me:
![image](https://user-images.githubusercontent.com/1659467/155569592-ab3d91fa-1163-4f44-af7c-ab99a9dfd1d3.png)

How long did you wait for the services to come up?

What are the full specs of your VM?

**Reply by mbaki** [reply]
On Centos 7 I'm having the same issue on 2.3.100  after an upgrade running on a standalone server

[root@iaiso minions]# so-status 

Checking Docker status

    Docker ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    

Checking container statuses

    so-aptcacherng -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-cortex ----------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-curator ---------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-dockerregistry ----------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-elastalert ------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-elasticsearch ---------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-filebeat --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-fleet ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ WAIT_START ]    
    so-grafana ---------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-idstools --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-influxdb --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-kibana ----------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-kratos ------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-logstash --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-mysql ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ WAIT_START ]    
    so-nginx ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ WAIT_START ]    
    so-playbook --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-redis ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ WAIT_START ]    
    so-sensoroni -------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-soc -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-soctopus --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-steno ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ WAIT_START ]    
    so-strelka-backend -------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-strelka-coordinator ---------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-strelka-filestream ----------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-strelka-frontend ------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-strelka-gatekeeper ----------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-strelka-manager -------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-suricata --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-telegraf --------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-thehive ---------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-thehive-es ------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]    
    so-wazuh ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ WAIT_START ]    
    so-zeek ------------------------------------------------------------------------------------------------------------------------------------------------------------- [ WAIT_START ]



[root@iaiso minions]# cat /root/soup.log 
### Preparing soup at Tue Mar  1 17:01:00 UTC 2022 ###

Checking to see if this is a manager.

This is a manager, so we can proceed.

There is an issue rendering the manager's pillars. Please correct the issues in the sls files mentioned below before running SOUP again.

"Rendering SLS 'minions.iaiso_standalone' failed. Please see master log for details."


salt-call pillar.get sensor:maininit
local:

salt-call grains.get ip_interfaces
local:
    ----------
    bond0:
    docker0:
        - 172.17.0.1
    em1:
        - 192.168.2.117
    enp0s20f0u5:
    lo:
        - 127.0.0.1
    veth1fc5c25:
    vethb0e8812:
    vethfa86723:
    wlp2s0:

salt-call pillar.get global:managerupdate
local:

[root@iaiso minions]# cat iaiso_standalone.sls
elasticsearch:
  es_port: 9200
  esclustername: '{{ grains.host }}'
  esheap: 5347m
  log_size_limit: 156
  mainint: em1
  mainip: 192.168.2.117
  node_route_type: hot
  node_type: ''
host:
  mainint: em1
 idstools:
   config:
     oinkcode: 'xxxxxxxxx'
     ruleset: ETPRO
     urls: null
  sids:
    disabled:
    - re:GPL
    - re:2010_
    - '2013031'
    - re:2011_
    - re:2012_
    - re:2013_
    - re:2014_
    - re:2015_
    - re:2016_
    - re:2009_
    - re:2017_
    - re:2018_
    - re:2034636
    - re:2033078
    - re:Cloudflare
    enabled:
    - re:Neshta
    - re:Neshuta
    - re:lockbit
    modify: null
kratos:
  kratoskey: xxxxxxxxxx
logstash_settings:
  ls_input_threads: 1
  ls_pipeline_batch_size: 125
  ls_pipeline_workers: 8
  lsheap: 1000m
manager:
  cur_close_days: 30
  domainstats: 0
  elastalert: 1
  es_port: 9200
  esclustername: '{{ grains.host }}'
  esheap: 5347m
  freq: 0
  grafana: 1
  mainint: em1
  mainip: 192.168.2.117
  mtu: 1500
  no_proxy: ''
  osquery: 0
  playbook: 0
  proxy: ''
  thehive: 0
ntp:
  servers:
  - 0.pool.ntp.org
  - 1.pool.ntp.org
patch:
  os:
    enabled: true
    schedule_name: manual
    source: direct
    splay: 300
sensor:
  hnsensor: null
  interface: bond0
  mainint: em1
  mainip: 192.168.2.117
  manager: ''
  mtu: 1500
  suriprocs: 1
  uniqueid: 1624119523
  zeek_lbprocs: 1
sensoroni:
  node_address: 192.168.2.117
  node_description: iaiso
soc:
  es_index_patterns: '*:so-*,*:endgame-*'
steno:
  enabled: true

**Reply by mbaki** [reply]
cat /opt/so/log/salt/master:

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/salt/pillar/__init__.py", line 892, in render_pstate
    **defaults
  File "/usr/lib/python3.6/site-packages/salt/template.py", line 99, in compile_template
    ret = render(input_data, saltenv, sls, **render_kwargs)
  File "/usr/lib/python3.6/site-packages/salt/loader/lazy.py", line 149, in __call__
    return self.loader.run(run_func, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/salt/loader/lazy.py", line 1201, in run
    return self._last_context.run(self._run_as, _func_or_method, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/contextvars/__init__.py", line 38, in run
    return callable(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/salt/loader/lazy.py", line 1216, in _run_as
    return _func_or_method(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/salt/renderers/yaml.py", line 68, in render
    raise SaltRenderError(exc)
salt.exceptions.SaltRenderError: while parsing a block mapping
  in "<unicode string>", line 1, column 1
did not find expected key
  in "<unicode string>", line 12, column 2
2022-03-01 21:10:40,666 [salt.pillar      :1220][CRITICAL][2271] Pillar render error: Rendering SLS 'minions.iaiso_standalone' failed. Please see master log for details.
2022-03-01 21:10:40,829 [salt.utils.job   :85  ][INFO    ][2270] Got return from iaiso_standalone for job 20220301211040828764
2022-03-01 21:10:40,829 [salt.utils.event :908 ][ERROR   ][2270] Event iteration failed with exception: 'list' object has no attribute 'items'
2022-03-01 21:10:40,832 [salt.utils.job   :85  ][INFO    ][2271] Got return from iaiso_standalone for job 20220301211040831355

**Comment by mbaki** [comment]
I went to https://yamlchecker.com/ and pasted my minion sls file and turns out it was a syntax error.

bad indentation of a mapping entry (12:2)

  9 |   node_type: ''
 10 | host:
 11 |   mainint: em1
 12 |  idstools:
-------^
 13 |    config:

**Reply by dougburks** [reply]
@mbaki It appears you have resolved your issue. In the future, please avoid replying to somebody else's discussion and start a separate discussion instead.
